samples/excel/85-preview-apis/comment.yaml

```diff
@@ -0,0 +1,167 @@
+order: 1
+id: excel-comment
+name: Comments
+description: 'Adds, edits, and removes comments and comment replies.'
+host: EXCEL
+api_set:
+    ExcelApi: 1.9
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); $("#addCommentToSelectedCell").click(() => tryCatch(addCommentToSelectedCell)); $("#addCommentToCell").click(() => tryCatch(addCommentToCell)); $("#addCommentReply").click(() => tryCatch(addCommentReply)); $("#getCommentMetadata").click(() => tryCatch(getCommentMetadata)); $("#editComment").click(() => tryCatch(editComment)); $("#editCommentReply").click(() => tryCatch(editCommentReply)); $("#deleteComment").click(() => tryCatch(deleteComment)); $("#deleteCommentReply").click(() => tryCatch(deleteCommentReply));
+        async function addCommentToSelectedCell() {
+            await Excel.run(async (context) => {
+                const selectedRange = context.workbook.getSelectedRange();
+                selectedRange.load(["rowIndex", "columnIndex"])
+                await context.sync();
+
+                // Note that an InvalidArgument error will be thrown if multiple cells are selected
+                const singleCell = selectedRange.getCell(selectedRange.rowIndex, selectedRange.columnIndex);
+                context.workbook.comments.add("TODO: add headers here.", singleCell);
+                await context.sync();
+            });
+        }
+        async function addCommentToCell() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Comments");
+                sheet.comments.add("TODO: add data.", "A2");
+                await context.sync();
+            });
+        }
+        async function addCommentReply() {
+            await Excel.run(async (context) => {
+                const comment = context.workbook.comments.getItemByCell("Comments!A1");
+                context.load(comment);
```

Comment:
I'd like to see a code comment explaining why the comment object has to be loaded before you can add to its replies collection. Also, `context.load(comment)` is equivalent to `comment.load()', so its an empty load that causes all of the comment's scalar properties to be loaded. Is that really necessary?